JavaScript/JavaScript.sublime-syntax

```diff
@@ -1362,7 +1356,7 @@ contexts:
   function-assignment:
     - match: |-
         (?x)(?=
-          (?:{{identifier}} \s* \. \s*)*
+          (?:{{identifier}} \s* {{dot_accessor}} \s*)*
```

Comment:
This is incorrect. Expression involving optional chaining cannot be a Lvalue.

Reply:
nice catch!